LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2020 Custobar Ltd
+Copyright (c) 2023 Custobar Ltd
```

Comment:
This is actually unnecessary. You only need to use the first publication year ont he license. See: https://hynek.me/til/copyright-years/

Not that it matters in any way here but just a FYI :)